pkg/xen-tools/initrd/eve-enter-container

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# PID of the started entrypoint container process.
+# PID file is created from the `init-initrd` script.
+pid=$(cat "/mnt/entrypoint-pid")
```

Comment:
I would add a check here, as in the `init-initrd` script the file creation failure is marked as non-critical.
```bash
if [ ! -f "/mnt/entrypoint-pid" ]; then
    echo "Error: PID file not found."
    exit 1
fi
```

Reply:
Chunk is added. Thanks.